### PR TITLE
Second part of the project. Works and About section added.

### PR DESCRIPTION
**Pull request by gandradep**
Grid was use to stack the different works and skill (About Me section).
The rest of the positioning was done with flexbox.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -20,42 +20,177 @@
...
+    <!-- works section -->
+
+    <section id="works">
+      <header class="padMobile">
+        <h1 class="title darkColor">My Recent Works</h1>
+        <hr>
+      </header>
+      </section>
+
+      <div class="gridContainer padMobile">
```
Comment (tufoinnkuo10) [inline index.html:68]: - [x] please ensure that the whole works section is inside a `section tag` of its own and the about section inside its own `section tag`.

File: index.html [hunk 2]
```diff
@@ -20,42 +20,177 @@
...
+          <a class="button" href="#">See Project</a>
+          </div>
+        </article>
+      </div>
+
+      <!-- About me section -->
+      <section class="aboutMe">
+        <header class="padMobile">
+          <h1 class="title darkColor">About Me</h1>
+          <p> I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded,
```
Comment (tufoinnkuo10) [inline index.html:151]: - [x] kindly add the font size to 20px like in the figma version.

File: index.html [hunk 3]
```diff
@@ -20,42 +20,177 @@
...
+          </ul>
+          <a class="button" href="#">See Project</a>
+          </div>
+        </article>
+      </div>
+
+      <!-- About me section -->
+      <section class="aboutMe">
+        <header class="padMobile">
+          <h1 class="title darkColor">About Me</h1>
```
Comment (tufoinnkuo10) [inline index.html:150]: - [x] please make the about me heading bolder

File: index.html [hunk 4]
```diff
@@ -20,42 +20,177 @@
...
+          <p> I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded,
+            don’t hestiate to contact me.
+          </p>
+          <a class="button" href="#">Get My Resume</a>
```
Comment (tufoinnkuo10) [inline index.html:154]: - [x] kindly make the button size bigger and more visible

File: index.html [hunk 5]
```diff
@@ -20,42 +20,177 @@
...
+        <div class="skillsGrid padMobile">
+          <div class="skills borderRad12">
+            <img src="assets/images/icon-languages.svg" alt="icon languages">
+              <article class="">
+                <h2 class="cardTitle">Languages</h2>
+                <ul class="greenColor">
+                  <li>JavaScript</li>
+                  <li>Ruby</li>
+                  <li>Html</li>
+                  <li>CSS</li>
```
Comment (tufoinnkuo10) [inline index.html:166]: - [x] please add the text sizes of your technologies. make them more visible

File: index.html [hunk 6]
```diff
@@ -20,42 +20,177 @@
...
+          <div class="skills borderRad12">
+            <img src="assets/images/icon-frameworks.svg" alt="icon frameworks">
+            <article class="">
+              <h2 class="cardTitle">Frameworks</h2>
+              <ul class="greenColor">
+                <li>Bootstrap</li>
+                <li>Ruby on rails</li>
+                <li>RSpec</li>
+                <li>Capybara</li>
+                <li>Selenium</li>
```
Comment (tufoinnkuo10) [inline index.html:179]: - [x] add their font sizes and make them more visible as well

File: index.html [hunk 7]
```diff
@@ -20,42 +20,177 @@
...
+          <div class="skills borderRad12">
+            <img src="assets/images/icon-skills.svg" alt="icon skills">
+            <article class="">
+              <h2 class="cardTitle">Skills</h2>
+              <ul class="greenColor">
+                <li>Codekit</li>
+                <li>Github</li>
+                <li>Codepen</li>
+                <li>Gitlab</li>
+                <li>Terminal</li>
```
Comment (tufoinnkuo10) [inline index.html:192]: - [x] please add their sizes too

[Figma ](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=48%3A988)version is as such
```
font-family: 'Inter';
font-style: normal;
font-weight: 600;
font-size: 12px;
line-height: 16px;
```

**Review by tufoinnkuo10 — CHANGES_REQUESTED** [review]
Hi @gandradep 

There are some issues that you still need to work on to go to the next project but you are almost there!

To Highlight
✅ You used the correct Github flow
✅ Descriptive pull request message
✅ linter error checks validated
✅ Documented your work professionally
✅ design well followed

Required Changes ♻️
Check the comments under the review

tufoin

Optional suggestions
_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using this [form.](https://airtable.com/shrQAqnBwek5a0O0s)

File: index.html (hunk 2 above)
Comment (gandradep) [reply index.html:151]: High @tufoinnkuo10 , last revision I didn't have problems with the font size, I'm using rem instead o px. Someone from Microverse mention we shouldn't copy exact the Figma, just make it look very similar. Please confirm me this before I do the changes .

**Review by ghost — APPROVED** [review]
Hi @gandradep , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

5SW


## Highlights

- Have used the right GitHub flow ✅
- Professional README.md file ✅
- Linter checks passed ✅


## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**